### PR TITLE
Add storage integ test to pipeline

### PR DESCRIPTION
**Pull request by Ashish-Nanda**
_Description of changes:_
Add storage integ test to pipeline

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -112,7 +112,7 @@ commands:
           name: 'Run cypress tests for << parameters.test_name >> Sample'
           command: |
             cd ~/amplify-js-samples-staging
-            yarn cypress:<< parameters.framework >> --spec "cypress/integration/<< parameters.spec >>"
+            sleep 30 && yarn cypress:<< parameters.framework >> --spec "cypress/integration/<< parameters.spec >>"
```
Comment (sammartinez) [inline .circleci/config.yml:115]: Why the sleep step prior ? Does it always fail when the server is spinning up to test? Is it intermitted?

File: .circleci/config.yml [hunk 2]
```diff
@@ -136,6 +136,7 @@ commands:
       - run:
           name: 'Install << parameters.test_name >> sample'
           command: |
+            cd ~/amplify-js-samples-staging-ui/samples
```
Comment (sammartinez) [inline .circleci/config.yml:139]: Is this needed in order to get to the correct directory?

File: .circleci/config.yml [hunk 3]
```diff
@@ -531,24 +552,3 @@ workflows:
             - build
           filters:
             <<: *releasable_branches
-      - deploy:
```
Comment (sammartinez) [inline .circleci/config.yml:534]: I believe we are trying to get it working is all but lets make sure prior to releasing this that this isnt removed.

**Comment by iartemiev** [comment]
Got this working with some changes to the sample.

See this run for all tests working: https://app.circleci.com/pipelines/github/aws-amplify/amplify-js/5306/workflows/0240984c-088e-4c17-8325-6ebc9f866b66

File: .circleci/config.yml (hunk 3 above)
Comment (iartemiev) [reply .circleci/config.yml]: 👍🏻 it's been restored

**Pull request merged**